Library/Homebrew/software_spec.rb

```diff
@@ -328,9 +328,6 @@ def select_download_strategy(specs)
 class BottleSpecification
   DEFAULT_PREFIX = HOMEBREW_DEFAULT_PREFIX
   DEFAULT_CELLAR = "#{DEFAULT_PREFIX}/Cellar".freeze
-  DEFAULT_DOMAIN_LINUX = "https://linuxbrew.bintray.com".freeze
-  DEFAULT_DOMAIN_MAC = "https://homebrew.bintray.com".freeze
-  DEFAULT_DOMAIN = ENV["HOMEBREW_BOTTLE_DOMAIN"].freeze || (OS.mac? ? DEFAULT_DOMAIN_MAC : DEFAULT_DOMAIN_LINUX)
```

Comment:
This'll be nice to get rid of!

Reply:
Sadly it's only moved to `brew.sh. =/ I agree that it would be nice to get rid of it.
https://github.com/Linuxbrew/brew/pull/744/files#diff-9252bdaacb30cce70aba1177ed9bc9baR272